doc/devel/contributing.rst

```diff
@@ -0,0 +1,422 @@
+.. _contributing:
+
+============
+Contributing
+============
+
+This project is a community effort, and everyone is welcome to
+contribute.
+
+The project is hosted on https://github.com/matplotlib/matplotlib
+
+Submitting a bug report
+=======================
+
+In case you experience issues using this package, do not hesitate to submit a
+ticket to the
+`Bug Tracker <https://github.com/matplotlib/matplotlib/issues>`_. You are
+also welcome to post feature requests or pull requests.
+
+
+Retrieving and installing the latest version of the code
+========================================================
+
+
+We use `Git <https://git-scm.com/>`_ for version control and
+`GitHub <https://github.com/>`_ for hosting our main repository.
+
+You can check out the latest sources with the command::
+
+    git clone git@github.com:matplotlib/matplotlib.git
+
+
+After obtaining a local copy of the matpotlib source code (:ref:`set-up-fork`),
+navigate to the matplotlib directory and run the following in the shell:
```

Comment:
Yes, it is definitely worth it.
I think it might be interesting to have a FAQ of common encountered problems with setting up working environments with matplotlib. This is one of the problem my students encountered, and the other problem being the tests massively failing.